script.js

```diff
@@ -2,57 +2,49 @@ let startBtn = document.querySelector("#startButton");
 startBtn.addEventListener("click", playGame);
 //ash says we can use this as a global object
 let gameState = ["", "", "", "", "", "", "", "", ""];
-function incrementCurrentPlayer(currentPlayer) {
-    currentPlayer++;
-    return currentPlayer;
-}
-function decideTurn(currentPlayer, playerMarker) {
-    if (currentPlayer % 2 == 0) {
-        playerMarker = 'o'
-    } else {
-        playerMarker = 'x'
+
+function decideTurn(currentPlayer) {
+    if (currentPlayer === 'o') {
+        return 'x';
+    }   else  {
+        return  'o';
     }
-    return playerMarker;
 }
-function addToScreen(tile, playerMarker) {
-    tile.innerHTML = playerMarker;
+function addToScreen(tile, currentPlayer) {
+    tile.innerHTML = currentPlayer;
 }
-function alterGameState(gameState, clicked, playerMarker) {
+function alterGameState(gameState, clicked, currentPlayer) {
     if(gameState[clicked] == "") {
-        gameState[clicked] = playerMarker;
+        gameState[clicked] = currentPlayer;
         return true;
     } else {
         return false;
     }
 }
+
 function playGame() {
     // this determines turn order
-    let currentPlayer = 1;
     // this variable holds alternating player symbols
-    let playerMarker;
+    let currentPlayer = 'x';
     const tiles = document.querySelectorAll('.tile');
     //this is a boolean to regulate whether a player can click a tile.
     let notClicked;
-    startBtn.removeEventListener('click', playGame, false)
+    startBtn.removeEventListener('click', playGame, false);
     startBtn.style.opacity = 0.5;
     tiles.forEach(function (tile) {
         tile.addEventListener('click', () => {
             //gets index value of tile from data attribute
             let clicked = tile.dataset.index;
             //assigns player marker based on currentPlayer value
-            playerMarker = decideTurn(currentPlayer, playerMarker);
-            let notClicked = alterGameState(gameState, clicked, playerMarker);
+            let notClicked = alterGameState(gameState, clicked, currentPlayer);
```

Comment:
This name `notClicked` is confusing me a bit - since if it returns true it means it was clicked? So notClicked being true means that it was clicked, which is breaking my brain.

Can we give this a better name?